pom.xml

```diff
@@ -37,15 +37,19 @@
         <opennms.tracker.version>0.7</opennms.tracker.version>
         <commons.io.version>2.11.0</commons.io.version>
         <commons.lang.version>2.6</commons.lang.version>
+        <grpc.version>1.47.0</grpc.version>
+        <protobuf.version>3.19.2</protobuf.version>
+        <protoc.version>21.0-rc-1</protoc.version>
```

Comment:
This is what's needed for MacOS 10.x

        <protobuf.version>3.21.2</protobuf.version>
        <protoc.version>3.20.1</protoc.version>

Reply:
Thanks, pulled it in!